### PR TITLE
Refine logging in pkg/dashboard/apiserver/event

### LINKED ISSUE #2874: Refine logging in `pkg/dashboard/apiserver/event`, move package level variable `log` into struct `Service` as a field.
> Please leave comments with your idea if you are interested in this issue. And comment `/assign` to operate self-assign before you work on it.

### PR DESCRIPTION
**Pull request by yoonian**
### What problem does this PR solve?

Close #2874

### What's changed and how it works?

Moved package level variable log into struct Service as a field. 

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.3
  - [ ] release-2.2
  - [ ] release-2.1

### Checklist

CHANGELOG

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### CODE REVIEW

**Comment by STRRL** [comment]
Welcome, @yoonian ! Thanks for your contribution!

File: pkg/dashboard/apiserver/event/fx.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"github.com/chaos-mesh/chaos-mesh/pkg/dashboard/core"
+)
+
+func Bootstrap(event core.EventStore,
+	workflowStore core.WorkflowStore,
+	conf *config.ChaosDashboardConfig,
+	logger logr.Logger,
+) *Service {
+	return NewService(event, workflowStore, conf, logger.WithName("events"))
+}
```
Comment (STRRL) [inline pkg/dashboard/apiserver/event/fx.go:31]: It seems the `Bootstrap` is exactly the same as the `NewService`. I think we do not require to use a new `Bootstrap` method.

**Review by STRRL — COMMENTED** [review]
rest LGTM! Thanks for your contribution!

File: pkg/dashboard/apiserver/event/fx.go (hunk 1 above)
Comment (yoonian) [reply pkg/dashboard/apiserver/event/fx.go:31]: Thanks, I'll apply.

**Comment by STRRL** [comment]
Welcom @yoonian , thanks for your contribution!

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Review by FingerLeader — APPROVED** [review]
LGTM!

**Comment by yoonian** [comment]
@Andrewmatilde Oops, I just force-pushed. Could you please proceed with the merge?

**Pull request merged**